book/src/api-reference/jsonrpc-api.md

```diff
@@ -21,7 +21,7 @@ To interact with a Solana node inside a JavaScript application, use the [solana-
 * [getClusterNodes](jsonrpc-api.md#getclusternodes)
 * [getEpochInfo](jsonrpc-api.md#getepochinfo)
 * [getEpochSchedule](jsonrpc-api.md#getepochschedule)
-* [getGenesisBlockhash](jsonrpc-api.md#getgenesisblockhash)
+* [getGenesisConfigHash](jsonrpc-api.md#getgenesisconfighash)
```

Comment:
I'd rather we keep this as ` getGenesisBlockhash`, and strike the downstream changes due to this rename.

Reply:
@mvines if we keep this as `getGenesisBlockhash` are we considering the genesis config's hash as a blockhash? Or is this being interpreted as the genesis block's hash? If it's the latter, I think we should return the hash of the first block here

Reply:
genesis config's hash is a blockhash as in bank0 that's the only recent_blockhash available to transactions to use.

Reply:
Changed to getGenesisHash as discussed in Discord